app/dao/provider_details_dao.py

```diff
@@ -80,25 +80,26 @@ def dao_reduce_sms_provider_priority(identifier, *, time_threshold):
     Will reduce a chosen sms provider's priority, and increase the other provider's priority by 10 points each.
     If either provider has been updated in the last `time_threshold`, then it won't take any action.
     """
-    amount_to_reduce_by = 10
+    # amount_to_reduce_by = 10
     providers_list = _get_sms_providers_for_update(time_threshold)
 
     if len(providers_list) < 2:
         current_app.logger.info("Not adjusting providers, number of active providers is less than 2.")
         return
 
-    providers = {provider.identifier: provider for provider in providers_list}
-    other_identifier = get_alternative_sms_provider(identifier)
-
-    reduced_provider = providers[identifier]
-    increased_provider = providers[other_identifier]
-
-    # always keep values between 0 and 100
-    reduced_provider_priority = max(0, reduced_provider.priority - amount_to_reduce_by)
-    increased_provider_priority = min(100, increased_provider.priority + amount_to_reduce_by)
-
-    _adjust_provider_priority(reduced_provider, reduced_provider_priority)
-    _adjust_provider_priority(increased_provider, increased_provider_priority)
+    # TODO right now we only have one provider.  Remove this?
+    # providers = {provider.identifier: provider for provider in providers_list}
+    # other_identifier = get_alternative_sms_provider(identifier)
+    #
+    # reduced_provider = providers[identifier]
+    # increased_provider = providers[other_identifier]
+    #
+    # # always keep values between 0 and 100
+    # reduced_provider_priority = max(0, reduced_provider.priority - amount_to_reduce_by)
+    # increased_provider_priority = min(100, increased_provider.priority + amount_to_reduce_by)
+    #
+    # _adjust_provider_priority(reduced_provider, reduced_provider_priority)
+    # _adjust_provider_priority(increased_provider, increased_provider_priority)
```

Comment:
Is this all related to the priority queue pieces from before?  If so then I agree, we should remove it.

Reply:
No, it's related to the app's concept have having two SMS providers, and toggling between them based on which one is performing better.

As I understand it, if 'firetext' starts throwing exceptions when messages are sent to it, more messages will get sent to 'mmg' and vice versa.   But we only have SNS, so it's impossible to hit one of these code blocks (the one that checks if number of providers is two or greater, and this code block that shifts the priority number around has no purpose in one provider implementation.

Reply:
Thanks for explaining!